portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,270 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+		integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+
+</head>
+
+<body>
+
+	<!-- navbar start -->
+	<nav class="navbar navbar-expand-lg navbar-light bg-dark mynav" id="home">
+		<div class="spinner-border text-primary" role="status">
+			<span class="sr-only">Loading...</span>
+		</div>
+		<div class="spinner-border text-secondary" role="status">
+			<span class="sr-only">Loading...</span>
+		</div>
+		<div class="spinner-border text-success" role="status">
+			<span class="sr-only">Loading...</span>
+		</div>
+
+		<a class="navbar-brand" href="#"></a>
+		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+		<div class="collapse navbar-collapse" id="navbarSupportedContent">
+			<ul class="navbar-nav mr-auto">
+				<li class="nav-item active">
+					<a class="nav-link" href="#home" id="colorchoice">Home <span class="sr-only">(current)</span></a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="#about" id="colorchoice">About</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="#achievements" id="colorchoice">Achievements</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="#community" id="colorchoice">Community</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="#skills" id="colorchoice">Skills</a>
+				</li>
+				<li class="nav-item active">
+					<a class="nav-link" href="#projects" id="colorchoice">Projects</a>
+				</li>
+				<li class="nav-item dropdown">
```

Comment:
I'm assuming you'll this functionality in next pull request

Reply:
Actually I had forgotten to remove this line of code